src/components/EmojiResultRow.css

```diff
@@ -0,0 +1,36 @@
+.component-emoji-result-row {
+  border-bottom: 1px solid #ccc;
+  padding: 10px;
+  height: 32px;
+  position: relative;
+  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+  cursor: pointer;
+}
+
+.component-emoji-result-row:hover {
+  background-color: #eee;
+}
+
+.component-emoji-result-row img {
```

Comment:
Never try to target element directly using html tag rather always try to create a new class to specify that element.

```css
.component-emoji-result-row img {
}
```
should become
```css
.component-emoji-result-row .my {
}
```